corehq/motech/repeaters/tasks.py

```diff
@@ -157,7 +157,7 @@ def _process_repeat_record(repeat_record):
             return
 
         try:
-            if repeat_record.repeater.is_paused or toggles.PAUSE_DATA_FORWARDING.enabled(repeat_record.domain):
+            if toggles.PAUSE_DATA_FORWARDING.enabled(repeat_record.domain):
```

Comment:
I think there are two pathways to this function. One is via `check_repeaters_in_partition()` but the other is `Repeater.register()`. I can't see where in this branch we catch that pathway to prevent sending to paused repeaters.

Reply:
Good point. Added it back in dd84e46f42d83e2c80fc99a916c4a5b96949d603, and then added the missing checks in ee26b2e06b83d984304c6044ab20cb0ee5af61e8. The reason I added it back is because I'll need to potentially account for a backlog of process_repeat_record tasks that could be associated with paused repeaters, so I'll give time for those to be processed before removing the check from `_process_repeat_record` entirely.